pkgs/build-support/setup-hooks/compress-man-pages.sh

```diff
@@ -21,6 +21,7 @@ compressManPages() {
 
     # Point symlinks to compressed manpages.
     find "$dir"/share/man/ -type l -a '!' -regex '.*\.\(bz2\|gz\)$' -print0 \
+        | sort -z
```

Comment:
I think this requires coreutils.

Reply:
That's correct. This script already uses `readlink` which also requires coreutils, so while I'm not sure where this dependency is specified I'm pretty sure it's already included.

Reply:
I thought it wasn't because the eval error mislead me.